{{ cookiecutter.__dirname }}/{% if cookiecutter.sphinx_docs in ['y', 'yes'] %}.readthedocs.yaml{% endif %}

```diff
@@ -9,7 +9,7 @@
 version: 2
 
 build:
-  os: ubuntu-20.04
+  os: ubuntu-latest
   tools:
     python: "3"
```

Comment:
I am curious if there is any difference between `"3"` and `"3.x"`.

Reply:
@dhalbert as far as I can tell "3.x" isn't listed as a valid choice https://docs.readthedocs.io/en/stable/config-file/v2.html#build-tools-python but the note for "3" calls it an `(alias for the latest 3.x version available on Read the Docs)` so it appears to function the same way as `3.x` does inside of github actions.yml files.